### PR TITLE
closes #143 Navitas R blog

### LINKED ISSUE #143: Blog Post: De-Mystifying R Programming in Clinical Trials
### Blog Post

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/AMRITA~1.SUR/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<link rel=Edit-Time-Data
href="file:///C:/Users/AMRITA~1.SUR/AppData/Local/Temp/msohtmlclip1/01/clip_editdata.mso">

<link rel=themeData
href="file:///C:/Users/AMRITA~1.SUR/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/AMRITA~1.SUR/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-IN style='tab-interval:36.0pt;word-wrap:break-word'>

<p class=MsoTitle><b><span lang=EN-US style='color:black;mso-color-alt:windowtext;
background:white'>De-Mystifying R Programming in Clinical Trials</span><span
lang=EN-US style='background:white'><o:p></o:p></span></b></p>

<p class=MsoNormal style='text-align:justify'><b><span lang=EN-US
style='mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin'>Venkatesan
Balu</span></b><span lang=EN-US style='mso-bidi-font-family:Calibri;mso-bidi-theme-font:
minor-latin'>, Associate Director, Global Data Sciences, Navitas Life Sciences<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>The use of R programming in clinical trials has
not been the most popular and obvious. Despite experiencing significant growth
in recent years, the adoption of R programming in clinical trials is not as
widespread and evident as anticipated. Practical implementation faces obstacles
due to various factors, including occasional misunderstandings, particularly in
the context of validation, and a notable lack of awareness regarding its
capabilities. However, despite these challenges, R is steadily establishing a
growing niche within the pharmaceutical industry.<o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Opportunities for R Programming in
Clinical Trials</span></b><b><span lang=EN-US style='font-size:16.0pt;
line-height:107%;color:windowtext;mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>Although R is versatile and applicable in various
settings, it is commonly associated with scientific computing and statistics.
In the context of clinical trials, where researchers aim to understand and
enhance drug development and testing processes, R has become a prominent tool
for analyzing the collected data. While SAS has been a longstanding programming
language for clinical trials, its high cost prompts the industry to explore
better alternatives. Therefore, there is a quest for sustainable technology and
tools that can effectively address industry challenges.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>To drive innovation, there is a need to move away
from traditional, inefficient processes and tools toward solutions that are
efficient, simple, easy to implement, reliable, and cost-effective.
Collaboration among industry stakeholders is crucial to develop a robust
technology ecosystem and establish consensus on validation and regulatory
benchmarks. Equally vital is preparing the workforce with the necessary
skillsets to meet future demands.<o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Current Usage Trends of R</span></b><b><span
lang=EN-US style='font-size:16.0pt;line-height:107%;color:windowtext;
mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:13.5pt;line-height:107%;mso-bidi-font-family:Calibri;
mso-bidi-theme-font:minor-latin;color:black;background:white'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>Analyzing the current trends of R in the
pharmaceutical industry reveals that its usage is currently below par in
activities related to Pharma Regulatory Submissions. However, R finds extensive
use in public health projects, healthcare economics, exploratory and scientific
analysis, trend identification, generating plots/graphs, specific statistical
analysis, and machine learning. R continues to advance steadily in clinical
trials, however lacks widespread usage within the clinical space. The notable
difference between SAS and R is that SAS is proprietary software, whereas R is
an open-source programming language.<o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>SAS or R Programming- Which is
Better?</span></b><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;
color:windowtext;mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;mso-bidi-font-family:Calibri;
mso-bidi-theme-font:minor-latin;color:black;background:white'>The ongoing
debate in the programming community revolves around whether to replace SAS with
R</span><span lang=EN-US style='font-size:12.0pt;line-height:107%;font-family:
"Times New Roman",serif;color:black;background:white'>, use both, or explore
other alternatives like Python. Instead of adopting an either-or scenario,
leveraging the strengths of each programming language for specific Data Science
problems is recommended, recognizing that one size does not fit all. Early adopters
of R have faced challenges, with regulatory compliance for R packages being a
common issue. For R to be considered for tasks related to regulatory submission,
a rigorous risk assessment of R packages, feasibility analysis, and the
establishment of processes for R usage through pilot projects with necessary
documentation become imperative.<o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Benefits of Using R Programming</span></b><b><span
lang=EN-US style='font-size:16.0pt;line-height:107%;color:windowtext;
mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>R, as a language and environment for statistical
computing and graphics, possesses characteristics that make it a potentially
powerful tool for Data Analysis. With approximately 2 million users worldwide
and three decades of legacy, R stands out as open-source software receiving
substantial support from the community. Its availability under the GNU General
Public License and extensive documentation contribute to its strength. R is
versatile, running on various platforms, offering a wide array of statistical
and graphical techniques, and its ease of producing publication-quality plots
enhances its appeal.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>The pharmaceutical industry has witnessed the
emergence of various R packages tailored for Clinical Trial Design, Monitoring,
and Analysis. Examples include Atable for creating tables for reporting
clinical trials, compareODM for comparing medical forms in CDISC ODM format,
CRTSize for sample size estimation in cluster randomized trials, and others.
These packages cater to different aspects of clinical trial data analysis,
showcasing the versatility of R in this domain. <o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>This article talks more about use of r in
clinical trials and how this will be used by taking advantages of open source
of R. <span style='mso-spacerun:yes'> </span>The FDA emphasizes the need for
fully documenting software packages used for statistical analysis in
submissions. The use of R poses specific challenges related to validation,
given its free and open-source nature. To address this, the R Foundation has released
guidance documents focusing on regulatory compliance, validation issues, and
the software development life cycle. However, this will not have problems in
implementing validation of the deliverables or any integration with the tools for
the statistical analysis. <o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Implementing Dual Programming</span></b><b><span
lang=EN-US style='font-size:16.0pt;line-height:107%;color:windowtext;
mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>As a procedural measure, we can implement dual
programming, where primary programmers focus on the SAS system for deliverables
while validators utilize an alternative program, such as R programming. As part
of demographic table validation, we generated the table using both SAS and R
programming.<o:p></o:p></span></p>

Table 01
--

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:13.5pt;line-height:107%;mso-bidi-font-family:Calibri;
mso-bidi-theme-font:minor-latin;color:black;background:white'><o:p>&nbsp;</o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Benefits of Using R Programming </span></b><b><span
lang=EN-US style='font-size:16.0pt;line-height:107%;color:windowtext;
mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>Given that the cost of the R-package is
non-chargeable, it can also serve as a potential tool for API integration. For
instance, in signal detection, R packages can prove to be valuable tools due to
the intricate derivation process for EBGM in the Bayesian approach, which aims
to mitigate false positive signals resulting from multiple comparisons. The
computation adjusts the observed-to-expected reporting ratio for temporal
trends and confounding variables such as age and sex. While both methods can
estimate this, the accessibility of R as free software enables easy integration
into any system as an API or for macro estimation purposes without any copyrights
issue. <o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Identifying the Limitations in Using
R- Programming</span></b><b><span lang=EN-US style='font-size:16.0pt;
line-height:107%;color:windowtext;mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>It is crucial to note that software cost is
essential to any company, including pharma and biotech ones. While R and
RStudio® are free and SAS® requires an annual license, using R instead of SAS
may not always lower costs. The cost of software is only one part of the
equation. To be used in a highly regulated industry such as pharmaceuticals,
software validation, maintenance, and support are critical, and their costs
need to be considered. Although R is free and open source, it comes with a
steep learning curve, lacks direct support from the company, and faces a
shortage of R programmers compared to those familiar with SAS®.<o:p></o:p></span></p>

<h2><b><span lang=EN-US style='font-size:16.0pt;line-height:107%;color:black;
mso-color-alt:windowtext;background:white'>Leveraging the Right Tools to
Capture Value</span></b><b><span lang=EN-US style='font-size:16.0pt;line-height:
107%;color:windowtext;mso-color-alt:windowtext;background:white'><o:p></o:p></span></b></h2>

<p class=MsoNormal><span lang=EN-US><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>Capturing the value of R programming starts with
a clear vision for its use and a systematic approach to identifying and
prioritizing the needs in the industry. Clinical Data Science is evolving
rapidly, and the industry actively seeks alternative solutions to unlock
valuable insights from diverse datasets. Recognizing the need for innovation,
collaboration, and efficient tools is crucial. Rather than viewing SAS, R, and
Python as mutually exclusive, leveraging the strengths of each for appropriate
Data Science problems provides a nuanced and effective approach. <o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif;
color:black;background:white'>Ensuring data quality, scientific integrity, and
regulatory compliance through risk assessment frameworks, validation, and
documentation are imperative in this dynamic landscape. The pharmaceutical
industry's journey toward embracing R reflects the broader trend of industries
recognizing the value and potential of open-source tools in addressing complex
challenges.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='mso-no-proof:yes'><![if !vml]>A person in a blue suit&#10;&#10;Description automatically generated<![endif]></span><span lang=EN-US style='font-size:
12.0pt;line-height:107%;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><b><span lang=EN-US
style='font-size:12.0pt;line-height:107%;font-family:"Times New Roman",serif'>Venkatesan
Balu</span></b><span lang=EN-US style='font-size:12.0pt;line-height:107%;
font-family:"Times New Roman",serif'> is the Associate Director, Global Data
Sciences, Navitas Life Sciences with 15+ years of experience in the
Biostatistics domain, and in Phase I to Phase IV Clinical Trials across various
therapeutic areas, BABE and PK studies. He has invaluable expertise in
providing inputs to study design, sample size, SAP, outlier evaluation, interim
analysis, complex statistical evaluation &amp; model selection, and regulatory
requirement. He is a technical leader in drug development strategy, adaptive
design, portfolio optimization, and decision-making in clinical trials.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN-US
style='mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin'><o:p>&nbsp;</o:p></span></p>

</body>

</html>

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: `c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
- [x] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [x] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@NavitasLifeSciences i made a draft PR with your blog converted to Quarto, and sent you write access so you can push updates to this blog.

Instead of me reviewing via the PR I just made updates directly to this draft blog for my comments as follows. Of course feel free to change anything back as you own this blog as author. Once you're happy we can tag others from the team to review this.

-	Added ® after any mention of SAS and removed mention of SAS “high cost” as such negative wording would not be suitable to place in a public blog in my opinion
-	Removed the dual programming section as don’t think it added any value for this blog to show that same table can be created in SAS or R
-	Added reference to pharmaverse as otherwise why post this blog on our site
-	I updated some of the wording such as R “usage is currently below par” as I don’t actually think it’s true – some of the blog reads more like a viewpoint from 2-3 years ago compared to the growing momentum we now see in R in our industry
-	Removed “The notable difference between SAS® and R is that SAS® is proprietary software, whereas R is an open-source programming language” as I feel readers would know this
-	Added reference to R consortium R submissions WG
-	I replaced the package examples given with pharmaverse examples as again this is a pharmaverse blog
-	Mentioned R Validation Hub rather than R Foundation for more specificity
-	Added a note on licensing
-	Learning curve of R removed “steep” and “lacks direct support from the company,” as those are not the case in many of the across-industry experiences I’ve seen
-	Added some images to help brighten the blog

**Comment by rossfarrugia** [comment]
Response from Navitas (Amrita Surendranath) by email:  _We are fine with the changes. Please go ahead with the next steps._

@bms63 @manciniedoardo want to take a look now? The spellcheck fails but I'm pretty sure I had added all those from this blog to wordlist so might be from other blogs, or I'm just missing something obvious...

**Comment by StefanThoma** [comment]
There is a short script in the R folder that will add all remaining words
to the spellcheck dictionary, for future reference :)

**Comment by rossfarrugia** [comment]
> There is a short script in the R folder that will add all remaining words to the spellcheck dictionary, for future reference :)

Odd as when I run the sections of `CICD.R` I get: 
`No spelling errors found.   `

So not sure why the check is failing,

**Comment by StefanThoma** [comment]
> > There is a short script in the R folder that will add all remaining words to the spellcheck dictionary, for future reference :)
> 
> Odd as when I run the sections of `CICD.R` I get: `No spelling errors found. `
> 
> So not sure why the check is failing,

There appears a strange error in our CICD workflow: 
`
Run testthat::test_that(desc = "No Typo", code = testthat::expect_equal(object = spelling::spell_check_files(list.files(pattern = ".*.qmd$", recursive = TRUE), ignore = readr::read_lines("inst/WORDLIST.txt")), expected = spelling::spell_check_files(path = "inst/WORDLIST.txt", ignore = readr::read_lines("inst/WORDLIST.txt"))))
── Failure (???): No Typo ──────────────────────────────────────────────────────
spelling::spell_check_files(...) not equal to spelling::spell_check_files(path = "inst/WORDLIST.txt", ignore = readr::read_lines("inst/WORDLIST.txt")).
Attributes: < Component “row.names”: Numeric: lengths (7, 0) differ >
Component “word”: Lengths ([7](https://github.com/pharmaverse/blog/actions/runs/8692333877/job/23836728476#step:7:8), 0) differ (string compare on first 0)
Component “found”: Length mismatch: comparison on first 0 components
`

@bms63 any ideas?

**Comment by bms63** [comment]
Not sure on this error.  Maybe there is a new r-libs spell check we could use?

File: posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+## Introduction
+
+The use of R programming in clinical trials has not always been the most
+popular and obvious in the past. Despite experiencing significant growth
+in recent years, the adoption of R programming in clinical trials is not
+as widespread and evident as anticipated. Practical implementation faces
+obstacles due to various factors, including occasional
+misunderstandings, particularly in the context of validation, and a
+notable lack of awareness regarding its capabilities. However, despite
+these challenges, R is steadily establishing a growing niche within the
```
Comment (manciniedoardo) [inline posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd:32]:
```suggestion
these challenges, R is steadily establishing a growing presence within the
```

File: posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+![SAS® or R?](Signpost.jpg){fig-align="center" width="500px"}
+
+The ongoing debate in the programming community revolves around whether
+to replace SAS® with R, use both, or explore other alternatives like
+Python. Instead of adopting an either-or scenario, leveraging the
+strengths of each programming language for specific Data Science
+problems is recommended, recognizing that one size does not fit all.
+Early adopters of R have faced challenges, with regulatory compliance
+for R packages being a common issue.
```
Comment (manciniedoardo) [inline posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd:78]: Can we counterbalance here with another phrase describing the successes that early adopters in R have had? This is an R blog after all

File: posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+General Public License and extensive documentation contribute to its
+strength. R is versatile, running on various platforms, offering a wide
+array of statistical and graphical techniques, and its ease of producing
+publication-quality plots enhances its appeal.
+
+The pharmaceutical industry has witnessed the emergence of various R
+packages tailored for Clinical Trial reporting. Examples include
+`{rtables}` for creating tables for reporting clinical trials,
+`{admiral}` for CDISC ADaM, `{pkglite}` to support eSubmission, and many
+others. These packages cater to different aspects of clinical trial data
```
Comment (manciniedoardo) [inline posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd:103]: Can we mention the pharmaverse here? Seems like a natural place to do so

File: posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+Ensuring data quality, scientific integrity, and regulatory compliance
+through risk assessment frameworks, validation, and documentation are
+imperative in this dynamic landscape. The pharmaceutical industry's
+journey toward embracing R reflects the broader trend of industries
+recognizing the value and potential of open-source tools in addressing
+complex challenges.
+
+![Leveraging the Right Tools](RightTool.png){fig-align="center"}
+
+## Author
```
Comment (manciniedoardo) [inline posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd:160]: we generally don't have an author section, please could we remove this?

File: posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+costs. The cost of software is only one part of the equation. To be used
+in a highly regulated industry such as pharmaceuticals, software
+validation, maintenance, and support are critical, and their costs need
+to be considered. Although R is free and open source, it comes with a
+learning curve, and in short term the industry might face a shortage of
+experienced pharma R programmers compared to those familiar with SAS®.
+
+## Leveraging the Right Tools to Capture Value
+
+Capturing the value of R programming starts with a clear vision for its
```
Comment (manciniedoardo) [inline posts/2024-04-15_de-_mystifying__.../de-_mystifying__r__programming_in__clinical__trials.qmd:141]: can we mention pharmaverse in the conclusion as well please?

**Review by manciniedoardo — CHANGES_REQUESTED** [review]

**Comment by manciniedoardo** [comment]
@NavitasLifeSciences thanks for your updates! Blog post is ready now 😄 

@bms63 @StefanThoma I guess we can merge once we find out what's going on with the spellcheck, or we can live dangerously and merge anyway... What do you think?

**Comment by NavitasLifeSciences** [comment]
Hi Edoardo,

Thank you so much! This is wonderful news.

Best Regards,
Amrita



Internal

**Comment by bms63** [comment]
If the Spelling is checked locally and is good, then lets just merge it.

**Comment by rossfarrugia** [comment]
As per Ben and Edoardo's messages i'm merging this one to close it out

**Pull request merged**